quinn-proto/src/connection/mod.rs

```diff
@@ -542,15 +652,16 @@ where
                 }
             }
 
+            // Don't increment space_idx.
+            // We stay in the current space and check if there is more data to send.
+        }
+
+        // Finish the last packet. This one might also need to be padded
+        if let Some(mut builder) = builder.take() {
```

Comment:
nit: Is `take()` needed here, or can we move out of `builder` completely?

Reply:
Right, we can move. We just need ´.take()` for the previous iterations to make the borrow checker happy. Same applies for `sent_frames`